addons/hr_holidays/tests/test_company_leave.py

```diff
@@ -327,7 +326,7 @@ def test_leave_whole_company_07(self):
         })
         company_leave._compute_date_from_to()
 
-        with self.assertQueryCount(__system__=607, admin=867):  # 770 community
+        with self.assertQueryCount(__system__=710, admin=867):  # 770 community
```

Comment:
This is a great jump :grimacing: we should at least have ideas where it is coming from